app/controllers/reminders_controller.rb

```diff
@@ -0,0 +1,76 @@
+class RemindersController < BasePublicController
+  before_action :create_journey_session_if_no_session
+
+  def show
+    @form = form_from_slug
+
+    render view_file
+  end
+
+  def update
+    @form = form_from_slug
+
+    if @form.valid?
+      redirect_to independent_reminder_path(
+        journey: journey::ROUTING_NAME,
+        slug: navigator.next_slug
+      )
+
+      @form.save!
+    else
+      render view_file
+    end
+  end
+
+  private
+
+  def create_journey_session_if_no_session
+    if session[journey_session_key].blank?
+      create_journey_session!
+    end
+  end
+
+  def create_journey_session!
+    journey_session = journey::SessionForm.create!(params)
+
+    session[journey_session_key] = journey_session.id
+
+    journey_session
+  end
+
+  def journey_session_key
+    :"#{journey::ROUTING_NAME}_journeys_session_id"
+  end
+
+  def view_file
+    params[:slug].underscore
+  end
```

Comment:
I think this is fine as we throw "form not found" if the slug isn't one we expect